Specs/NumberWithUnit/English/CurrencyModel.json

```diff
@@ -1,1441 +1,1525 @@
-[{
+[
+ {
+  "Input": "it only cost 3 dollars 50 cents.",
```

Comment:
When you add new tests, please add at the end of the Spec. It makes it easier to keep specs in sync between languages.